packages/e2e/cypress/e2e/app/settings/profile.cy.ts

```diff
@@ -15,6 +15,18 @@ describe('Settings - Profile', () => {
             },
         });
     });
+    afterEach(() => {
+        // Reset to default values
+        cy.request({
```

Comment:
This was also failing a lot, fixed it here

Reply:
Nice. Unrelated improvement?